.github/workflows/check-amdllpc-docker.yml

```diff
@@ -49,4 +51,18 @@ jobs:
                             --build-arg LLPC_REPO_NAME="${GITHUB_REPOSITORY}"
                             --build-arg LLPC_REPO_REF="${GITHUB_REF}"
                             --build-arg LLPC_REPO_SHA="${GITHUB_SHA}"
+                            --build-arg FEATURES="${{ matrix.feature-set }}"
                             --tag llpc/ci-shaderdb
+      - name: Copy code coverage report to host and upload to GCS
+        if: contains(matrix.feature-set, '+coverage') && github.event.pull_request.number
+        run: |
+          docker run -v "$HOME:/host" llpc/ci-shaderdb 'bash' '-c' 'cp -r /vulkandriver/coverage_report /host/ && chmod -R 777 /host/coverage_report'
```

Comment:
Why `777`? Are permissions the issue or the file owner? If the former, could the generation script or docker step change the permissions?  If the latter, could we `chown` instead?

Reply:
The owner is wrong. I changed it to `chown`.